src/metric-engine/src/metadata_region.rs

```diff
@@ -76,11 +78,34 @@ impl MetadataRegion {
         }
     }
 
-    pub async fn open_logical_region(&self, logical_region_id: RegionId) {
+    /// Open a logical region.
+    ///
+    /// Returns true if the logical region is opened for the first time.
+    pub async fn open_logical_region(&self, logical_region_id: RegionId) -> bool {
+        info!("open_logical_region: {:?}", logical_region_id);
```

Comment:
It's better to move it to the end and log the return value (true or false).

Reply:
Ooops, I just forgot to remove the log here.